website/blog/2024-01-24-running-tabby-locally-with-rocm/index.md

```diff
@@ -1,10 +1,11 @@
 ---
-slug: running-tabby-locally-with-rocm.md
 title: Running Tabby Locally with AMD ROCm
 authors: [boxbeam]
 tags: [deployment]
 ---
 
+⚠️ ROCm is only supported in Tabby's nightly builds! Support will be stable in version 0.8.
```

Comment:
add a info block https://docusaurus.io/docs/next/markdown-features/admonitions

Reply:
(also make nightly text a link to nightly release page on github)

Reply:
GitHub uses a different syntax for their extension to markdown, I've used that

Reply:
but we're using docusaurus to render the blog post 👀